### PR TITLE
Remove the sticky example and add BookedIN example

### PR DESCRIPTION
**Pull request by olafleur**
The sticky sample is 404 not found.

[Review on Reviewable](https://reviewable.io/reviews/gwtproject/gwt-site/118)

### CODE REVIEW

**Comment by olafleur** [comment]
I signed it.

**Comment by olafleur** [comment]
screen shot 2015-09-03 at 1 21 48 pm

**Comment by olafleur** [comment]
@jDramaix or @manolo : Any idea when this will be merged?

File: src/main/markdown/examples.md [hunk 1]
```diff
@@ -207,6 +191,19 @@ was built using GWT. Use the free trial to see it in action.</div>
...
 <div class="project">
   <div class="screenshot">
+    <a href="https://getbookedin.com/?cid=256" target="_blank"><img src="images/bookedin.png" alt="BookedIN"/></a>
+  </div>
+  <div class="info">
+    <div class="name"><a href="https://getbookedin.com/?cid=256" target="_blank">BookedIN</a></div>
+    <div class="snippet">BookedIN is an online appointment booking, scheduling and payment system for small business.
+      Customers can book and pay for appointments online, and the business owner can automate their internal booking
+      process, reducing their administrative workload. BookedIN uses <a href="https://gwtp.arcbees.com">GWTP</a>
+      which has helped to simplify and accelerate development. </div>
```
Comment (jDramaix) [inline src/main/markdown/examples.md]: I would remove this line. We don't want to advertise any third-party framework in the gwt website. You can mention which technology is used but that's it

**Comment by olafleur** [comment]
I've made the modification @jDramaix

**Comment by jDramaix** [comment]
Thanks a lot

**Comment by olafleur** [comment]
What's the frequency of deployment of the website? I don't see the changes reflected on the website for the moment.

**Comment by jDramaix** [comment]
it's a manual process.

**Comment by olafleur** [comment]
Ok. No problem. ;)

**Pull request merged**